tests/fixtures/__init__.py

```diff
@@ -14,7 +14,3 @@
 # limitations under the License.
 
 """Fixtures that can be used in both unit and integration tests"""
```

Comment:
Since we stopped differentiating between unit and integration tests.

```suggestion
"""Fixtures that can be used in tests"""
```